app/routes/events/components/EventEditor/index.tsx

```diff
@@ -125,10 +147,91 @@ function EventEditor({
           name="cover"
           component={ImageUploadField}
           uploadFile={uploadFile}
-          edit={isEditPage && ((token) => setCoverPhoto(eventId, token))}
           aspectRatio={20 / 6}
-          img={event.cover}
+          img={useImageGallery ? imageGalleryUrl : event.cover}
         />
+        <Modal
+          show={showImageGallery}
+          onHide={() => setShowImageGallery(false)}
+          contentClassName={styles.imageGallery}
+        >
+          <>
+            <Flex column>
+              <h1>Bildegalleri</h1>
+              <Flex wrap alignItems="center" justifyContent="space-around">
+                {imageGallery &&
+                  imageGallery.map((e) => {
+                    return (
+                      <div key={e.key} className={styles.imageGalleryContainer}>
+                        <ConfirmModalWithParent
```

Comment:
The confirm modal is currently implemented differently. Take a look at use cases of `ConfirmModal`